[BUG] User registration fails if email is provided

### Description

I am running paperless with Docker, using traefik as a reverse proxy. I have Mailjet configured as the email server and it works: it sends out the password reset email correctly.

However, when I register a new user with an email, I get the following error:

```
django.urls.exceptions.NoReverseMatch: Reverse for 'account_confirm_email' not found. 'account_confirm_email' is not a valid view function or pattern name.
```

I tried to change the `PAPERLESS_ACCOUNT_EMAIL_VERIFICATION` from `optional` to `mandatory` but I still get the same error.

### Steps to reproduce

1. Register a new user providing an email

### Webserver logs

```bash
webserver-1  | Internal Server Error: /accounts/signup/
webserver-1  | Traceback (most recent call last):
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/asgiref/sync.py", line 518, in thread_handler
webserver-1  |     raise exc_info[1]
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/django/core/handlers/exception.py", line 42, in inner
webserver-1  |     response = await get_response(request)
webserver-1  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/asgiref/sync.py", line 518, in thread_handler
webserver-1  |     raise exc_info[1]
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/django/core/handlers/base.py", line 253, in _get_response_async
webserver-1  |     response = await wrapped_callback(
webserver-1  |                ^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/asgiref/sync.py", line 468, in __call__
webserver-1  |     ret = await asyncio.shield(exec_coro)
webserver-1  |           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/asgiref/current_thread_executor.py", line 40, in run
webserver-1  |     result = self.fn(*self.args, **self.kwargs)
webserver-1  |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/asgiref/sync.py", line 522, in thread_handler
webserver-1  |     return func(*args, **kwargs)
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/django/views/generic/base.py", line 104, in view
webserver-1  |     return self.dispatch(request, *args, **kwargs)
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/django/utils/decorators.py", line 48, in _wrapper
webserver-1  |     return bound_method(*args, **kwargs)
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/allauth/decorators.py", line 12, in wrap
webserver-1  |     resp = function(request, *args, **kwargs)
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/django/utils/decorators.py", line 48, in _wrapper
webserver-1  |     return bound_method(*args, **kwargs)
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/django/utils/decorators.py", line 48, in _wrapper
webserver-1  |     return bound_method(*args, **kwargs)
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/django/views/decorators/debug.py", line 143, in sensitive_post_parameters_wrapper
webserver-1  |     return view(request, *args, **kwargs)
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/django/utils/decorators.py", line 48, in _wrapper
webserver-1  |     return bound_method(*args, **kwargs)
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/django/views/decorators/cache.py", line 80, in _view_wrapper
webserver-1  |     response = view_func(request, *args, **kwargs)
webserver-1  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/allauth/account/views.py", line 158, in dispatch
webserver-1  |     return super().dispatch(request, *args, **kwargs)
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/django/utils/decorators.py", line 48, in _wrapper
webserver-1  |     return bound_method(*args, **kwargs)
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/django/utils/decorators.py", line 48, in _wrapper
webserver-1  |     return bound_method(*args, **kwargs)
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/django/views/decorators/cache.py", line 80, in _view_wrapper
webserver-1  |     response = view_func(request, *args, **kwargs)
webserver-1  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/allauth/account/mixins.py", line 53, in dispatch
webserver-1  |     response = super().dispatch(request, *args, **kwargs)
webserver-1  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/allauth/account/mixins.py", line 118, in dispatch
webserver-1  |     return super().dispatch(request, *args, **kwargs)
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/django/views/generic/base.py", line 143, in dispatch
webserver-1  |     return handler(request, *args, **kwargs)
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/allauth/account/mixins.py", line 82, in post
webserver-1  |     response = self.form_valid(form)
webserver-1  |                ^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/allauth/account/views.py", line 169, in form_valid
webserver-1  |     return flows.signup.complete_signup(
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/allauth/account/internal/flows/signup.py", line 72, in complete_signup
webserver-1  |     return perform_login(request, login)
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/allauth/account/internal/flows/login.py", line 86, in perform_login
webserver-1  |     return resume_login(request, login)
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/allauth/account/internal/flows/login.py", line 95, in resume_login
webserver-1  |     response = ctrl.handle()
webserver-1  |                ^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/allauth/account/stages.py", line 106, in handle
webserver-1  |     response, cont = stage.handle()
webserver-1  |                      ^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/allauth/account/stages.py", line 140, in handle
webserver-1  |     send_email_confirmation(
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/allauth/account/utils.py", line 284, in send_email_confirmation
webserver-1  |     return flows.email_verification.send_verification_email(
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/allauth/account/internal/flows/email_verification.py", line 214, in send_verification_email
webserver-1  |     email_address.send_confirmation(request, signup=signup)
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/allauth/account/models.py", line 107, in send_confirmation
webserver-1  |     confirmation.send(request, signup=signup)
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/allauth/account/models.py", line 136, in send
webserver-1  |     get_adapter().send_confirmation_mail(request, self, signup)
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/allauth/account/adapter.py", line 637, in send_confirmation_mail
webserver-1  |     "activate_url": self.get_email_confirmation_url(
webserver-1  |                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/allauth/account/adapter.py", line 607, in get_email_confirmation_url
webserver-1  |     return flows.email_verification.get_email_verification_url(
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/allauth/account/internal/flows/email_verification.py", line 94, in get_email_verification_url
webserver-1  |     url = reverse("account_confirm_email", args=[emailconfirmation.key])
webserver-1  |           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/django/urls/base.py", line 88, in reverse
webserver-1  |     return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/django/urls/resolvers.py", line 831, in _reverse_with_prefix
webserver-1  |     raise NoReverseMatch(msg)
webserver-1  | django.urls.exceptions.NoReverseMatch: Reverse for 'account_confirm_email' not found. 'account_confirm_email' is not a valid view function or pattern name.
webserver-1  | [2025-02-17 21:43:04,759] [ERROR] [django.request] Internal Server Error: /accounts/signup/
webserver-1  | Traceback (most recent call last):
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/asgiref/sync.py", line 518, in thread_handler
webserver-1  |     raise exc_info[1]
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/django/core/handlers/exception.py", line 42, in inner
webserver-1  |     response = await get_response(request)
webserver-1  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/asgiref/sync.py", line 518, in thread_handler
webserver-1  |     raise exc_info[1]
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/django/core/handlers/base.py", line 253, in _get_response_async
webserver-1  |     response = await wrapped_callback(
webserver-1  |                ^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/asgiref/sync.py", line 468, in __call__
webserver-1  |     ret = await asyncio.shield(exec_coro)
webserver-1  |           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/asgiref/current_thread_executor.py", line 40, in run
webserver-1  |     result = self.fn(*self.args, **self.kwargs)
webserver-1  |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/asgiref/sync.py", line 522, in thread_handler
webserver-1  |     return func(*args, **kwargs)
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/django/views/generic/base.py", line 104, in view
webserver-1  |     return self.dispatch(request, *args, **kwargs)
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/django/utils/decorators.py", line 48, in _wrapper
webserver-1  |     return bound_method(*args, **kwargs)
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/allauth/decorators.py", line 12, in wrap
webserver-1  |     resp = function(request, *args, **kwargs)
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/django/utils/decorators.py", line 48, in _wrapper
webserver-1  |     return bound_method(*args, **kwargs)
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/django/utils/decorators.py", line 48, in _wrapper
webserver-1  |     return bound_method(*args, **kwargs)
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/django/views/decorators/debug.py", line 143, in sensitive_post_parameters_wrapper
webserver-1  |     return view(request, *args, **kwargs)
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/django/utils/decorators.py", line 48, in _wrapper
webserver-1  |     return bound_method(*args, **kwargs)
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/django/views/decorators/cache.py", line 80, in _view_wrapper
webserver-1  |     response = view_func(request, *args, **kwargs)
webserver-1  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/allauth/account/views.py", line 158, in dispatch
webserver-1  |     return super().dispatch(request, *args, **kwargs)
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/django/utils/decorators.py", line 48, in _wrapper
webserver-1  |     return bound_method(*args, **kwargs)
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/django/utils/decorators.py", line 48, in _wrapper
webserver-1  |     return bound_method(*args, **kwargs)
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/django/views/decorators/cache.py", line 80, in _view_wrapper
webserver-1  |     response = view_func(request, *args, **kwargs)
webserver-1  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/allauth/account/mixins.py", line 53, in dispatch
webserver-1  |     response = super().dispatch(request, *args, **kwargs)
webserver-1  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/allauth/account/mixins.py", line 118, in dispatch
webserver-1  |     return super().dispatch(request, *args, **kwargs)
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/django/views/generic/base.py", line 143, in dispatch
webserver-1  |     return handler(request, *args, **kwargs)
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/allauth/account/mixins.py", line 82, in post
webserver-1  |     response = self.form_valid(form)
webserver-1  |                ^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/allauth/account/views.py", line 169, in form_valid
webserver-1  |     return flows.signup.complete_signup(
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/allauth/account/internal/flows/signup.py", line 72, in complete_signup
webserver-1  |     return perform_login(request, login)
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/allauth/account/internal/flows/login.py", line 86, in perform_login
webserver-1  |     return resume_login(request, login)
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/allauth/account/internal/flows/login.py", line 95, in resume_login
webserver-1  |     response = ctrl.handle()
webserver-1  |                ^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/allauth/account/stages.py", line 106, in handle
webserver-1  |     response, cont = stage.handle()
webserver-1  |                      ^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/allauth/account/stages.py", line 140, in handle
webserver-1  |     send_email_confirmation(
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/allauth/account/utils.py", line 284, in send_email_confirmation
webserver-1  |     return flows.email_verification.send_verification_email(
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/allauth/account/internal/flows/email_verification.py", line 214, in send_verification_email
webserver-1  |     email_address.send_confirmation(request, signup=signup)
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/allauth/account/models.py", line 107, in send_confirmation
webserver-1  |     confirmation.send(request, signup=signup)
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/allauth/account/models.py", line 136, in send
webserver-1  |     get_adapter().send_confirmation_mail(request, self, signup)
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/allauth/account/adapter.py", line 637, in send_confirmation_mail
webserver-1  |     "activate_url": self.get_email_confirmation_url(
webserver-1  |                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/allauth/account/adapter.py", line 607, in get_email_confirmation_url
webserver-1  |     return flows.email_verification.get_email_verification_url(
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/allauth/account/internal/flows/email_verification.py", line 94, in get_email_verification_url
webserver-1  |     url = reverse("account_confirm_email", args=[emailconfirmation.key])
webserver-1  |           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/django/urls/base.py", line 88, in reverse
webserver-1  |     return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
webserver-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/django/urls/resolvers.py", line 831, in _reverse_with_prefix
webserver-1  |     raise NoReverseMatch(msg)
webserver-1  | django.urls.exceptions.NoReverseMatch: Reverse for 'account_confirm_email' not found. 'account_confirm_email' is not a valid view function or pattern name.
```

### Browser logs

```bash

```

### Paperless-ngx version

2.14.7

### Host OS

Ubuntu 24.04

### Installation method

Docker - official image

### System status

```json
{
    "pngx_version": "2.14.7",
    "server_os": "Linux-6.8.0-50-generic-x86_64-with-glibc2.36",
    "install_type": "docker",
    "storage": {
        "total": 206900281344,
        "available": 136805658624
    },
    "database": {
        "type": "postgresql",
        "url": "flawlens",
        "status": "OK",
        "error": null,
        "migration_status": {
            "latest_migration": "documents.1063_alter_workflowactionwebhook_url",
            "unapplied_migrations": []
        }
    },
    "tasks": {
        "redis_url": "redis://broker:6379",
        "redis_status": "OK",
        "redis_error": null,
        "celery_status": "OK",
        "index_status": "OK",
        "index_last_modified": "2025-02-17T14:18:42.277927+01:00",
        "index_error": null,
        "classifier_status": "OK",
        "classifier_last_trained": null,
        "classifier_error": null
    }
}
```

### Browser

Firefox

### Configuration changes

`docker-compose.yml`

```yaml
services:
    traefik:
       # ... traefik config

    broker:
        image: docker.io/library/redis:7
        restart: unless-stopped
        volumes:
            - redisdata:/data
        networks:
            - internal

    db:
        # ... default config
        networks:
            - internal

    webserver:
        image: ghcr.io/paperless-ngx/paperless-ngx:latest
        restart: unless-stopped
        depends_on:
            - db
            - broker
        volumes:
            - data:/usr/src/paperless/data
            - media:/usr/src/paperless/media
            - ./export:/usr/src/paperless/export
            - ./consume:/usr/src/paperless/consume
        env_file: docker-compose.env
        environment:
            PAPERLESS_REDIS: redis://broker:6379
            PAPERLESS_DBHOST: db
        networks:
            - internal
            - web
        labels:
             # ... traefik-related labels

networks:
    web:
        external: true
    internal:
        name: paperless_internal
        external: false

volumes:
    data:
    media:
    pgdata:
    redisdata:
```

----

`docker-compose.env`:

```env
PAPERLESS_DEBUG='YES'

USERMAP_UID=0
USERMAP_GID=0
PAPERLESS_SECRET_KEY=<my secret>

# Hosting
PAPERLESS_URL=https://<my custom domain>
USE_X_FORWARD_HOST=true
USE_X_FORWARD_PORT=true

PAPERLESS_ACCOUNT_ALLOW_SIGNUPS=true
PAPERLESS_ACCOUNT_EMAIL_VERIFICATION=optional # optional | mandatory | none


PAPERLESS_EMAIL_HOST=in.mailjet.com
PAPERLESS_EMAIL_PORT=2525
PAPERLESS_EMAIL_HOST_USER= <mailjet api key>
PAPERLESS_EMAIL_HOST_PASSWORD= <mailjet token>
PAPERLESS_EMAIL_FROM= <custom sender address>
PAPERLESS_EMAIL_USE_TLS=true
```

### Please confirm the following

- [x] I believe this issue is a bug that affects all users of Paperless-ngx, not something specific to my installation.
- [x] This issue is not about the OCR or archive creation of a specific file(s). Otherwise, please see above regarding OCR tools.
- [x] I have already searched for relevant existing issues and discussions before opening this report.
- [x] I have updated the title field above with a concise description.

--- a/src/paperless/urls.py
+++ b/src/paperless/urls.py
@@ -306,6 +306,11 @@ urlpatterns = [
                     ),
                 ),
                 re_path(
+                    r"^confirm-email/(?P<key>[-:\w]+)/$",
+                    allauth_account_views.ConfirmEmailView.as_view(),
+                    name="account_confirm_email",
+                ),
+                re_path(
                     r"^password/reset/key/(?P<uidb36>[0-9A-Za-z]+)-(?P<key>.+)/$",
                     allauth_account_views.password_reset_from_key,
                     name="account_reset_password_from_key",

